actions/drain_node_handler.go

```diff
@@ -352,6 +354,20 @@ func (h *drainNodeHandler) deletePod(ctx context.Context, options metav1.DeleteO
 	return nil
 }
 
+func (h *drainNodeHandler) logCastPodsToEvict(castPods []v1.Pod) {
+	if len(castPods) == 0 {
+		return
+	}
+
+	castPodsNames := make([]string, 0, len(castPods))
+	for _, p := range castPods {
+		castPodsNames = append(castPodsNames, p.Name)
+	}
+	joinedPodNames := strings.Join(castPodsNames, ", ")
+
+	logrus.Warnf("evicting CAST AI pods: %s", joinedPodNames)
```

Comment:
I think we will not have all fields from here, right?
https://github.com/castai/cluster-controller/blob/main/main.go#L129-L134

Reply:
we already have something else in the handler, lots of fields
https://github.com/castai/cluster-controller/blob/main/actions/drain_node_handler.go#L82

Reply:
The main idea is just know how often we are killing castware and cross logs with other ones to debug. We have the node_id and other information from the handler.

Reply:
Will this log have all fields from main.go or not?  If not it will not have a context of the controller.